### PR TITLE
build: Ellipsis.dev and Dockerfile for Pretzel

### PR DESCRIPTION
**Pull request by khareyash05**
setup dockerfile for the project

----

| <a href="https://ellipsis.dev" target="_blank">Ellipsis</a> | :rocket: This PR description was created by [Ellipsis](https://www.ellipsis.dev) for commit 10d54962bb6a3d97fb178876609adb69efad7269.  | 
|--------|--------|

### Summary:
This PR introduces Docker support for the Pretzel AI project and adds a configuration file for Ellipsis.dev.

**Key points**:
- Added Dockerfile for building a Docker image of the project
- Added .dockerignore to exclude unnecessary files from Docker context
- Added ellipsis.yaml for Ellipsis.dev configuration

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

**Comment by khareyash05** [comment]
@ramonverse This is a general Dockerfile we can use while setting up the project as well as can be used for setting up the CI pipeline as well

**Comment by ramonverse** [comment]
@ellipsis-dev this PR is missing ellipsis.yaml can you fix that

**Comment by ramonverse** [comment]
@khareyash05 we need to add the yaml file here too right? https://docs.ellipsis.dev/config

**Comment by khareyash05** [comment]
> @khareyash05 we need to add the yaml file here too right? https://docs.ellipsis.dev/config

Yes I have a PR on my fork. I am testing out changes. This is just the general dockerfile we can use for setup not related to ellipsis

**Comment by khareyash05** [comment]
https://github.com/khareyash05/pretzelai/pull/1

**Comment by khareyash05** [comment]
@ellipsis-dev Review

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /app
+
+COPY package*.json .
+
+RUN npm install
+
+COPY . .
+
+CMD ["npm","start"]
```
[the remark this answers is not in the document]
Comment (khareyash05) [reply Dockerfile:11]: @ellipsis-dev fix it

**Comment by khareyash05** [comment]
@ellipsis-dev review

**Pull request merged**